tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -149,6 +167,68 @@ func NewStresser(s string, sc *stressConfig, m *member) Stresser {
 			keysPerLease: sc.keysPerLease,
 			rateLimiter:  sc.rateLimiter,
 		}
+	case "election-runner":
+		reqRate := 100
+		sc.rateLimiter.SetLimit(sc.rateLimiter.Limit() - rate.Limit(reqRate))
+		return &runnerStresser{
+			cmdStr: sc.etcdRunnerPath,
+			args: []string{
+				"election",
+				string(time.Now().UnixNano()), // election name as current nano time
+				"--dial-timeout=10s",
+				"--endpoints", m.grpcAddr(),
+				"--total-client-connections=10",
+				"--rounds=0", // runs forever
+				"--req-rate", fmt.Sprintf("%v", reqRate),
+			},
+			errc:  make(chan error, 1),
+			donec: make(chan struct{}),
+		}
+	case "watch-runner":
+		reqRate := 100
+		sc.rateLimiter.SetLimit(sc.rateLimiter.Limit() - rate.Limit(reqRate))
+		return &runnerStresser{
```

Comment:
this should have a constructor `newRunnerStresser(args, sc, reqRate)` since there's repeated setup for each runner command